WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -0,0 +1,685 @@
+    }
+    
+    // MARK: - Configuration
+    
+    override func heightForFooterView() -> CGFloat {
+        return self.dynamicType.postListHeightForFooterView
+    }
+    
+    private func configureCellsForLayout() {
+        
+        let bundle = NSBundle.mainBundle()
+        
+        textCellForLayout = bundle.loadNibNamed(self.dynamicType.postCardTextCellNibName, owner: nil, options: nil)[0] as! PostCardTableViewCell
+        forceUpdateCellLayout(textCellForLayout)
+        
+        imageCellForLayout = bundle.loadNibNamed(self.dynamicType.postCardImageCellNibName, owner: nil, options: nil)[0] as! PostCardTableViewCell
+        forceUpdateCellLayout(imageCellForLayout)
+    }
+    
+    private func configureTableView() {
+        
+        assert(tableView != nil, "We expect tableView to never be nil at this point.")
+        
+        guard let tableView = tableView else {
+            return
+        }
+        
+        tableView.accessibilityIdentifier = "PostsTable"
+        tableView.isAccessibilityElement = true
+        tableView.separatorStyle = .None
+        
+        let bundle = NSBundle.mainBundle()
+        
+        // Register the cells
+        let postCardTextCellNib = UINib(nibName: self.dynamicType.postCardTextCellNibName, bundle: bundle)
+        tableView.registerNib(postCardTextCellNib, forCellReuseIdentifier: self.dynamicType.postCardTextCellIdentifier)
+        
+        let postCardImageCellNib = UINib(nibName: self.dynamicType.postCardImageCellNibName, bundle: bundle)
+        tableView.registerNib(postCardImageCellNib, forCellReuseIdentifier: self.dynamicType.postCardImageCellIdentifier)
+        
+        let postCardRestoreCellNib = UINib(nibName: self.dynamicType.postCardRestoreCellNibName, bundle: bundle)
+        tableView.registerNib(postCardRestoreCellNib, forCellReuseIdentifier: self.dynamicType.postCardRestoreCellIdentifier)
+    }
+    
+    private func noResultsTitleText() -> String {
+        if syncHelper?.isSyncing == true {
+            return NSLocalizedString("Fetching posts...", comment: "A brief prompt shown when the reader is empty, letting the user know the app is currently fetching new posts.");
+        }
+        
+        if let filter = currentPostListFilter() {
+            let titles = noResultsTitles()
+            let title = titles[filter.filterType]
+            return title ?? ""
+        } else {
+            return ""
+        }
+    }
+    
+    private func noResultsTitles() -> [PostListStatusFilter:String] {
+        if isSearching() {
+            return noResultsTitlesWhenSearching()
+        } else {
+            return noResultsTitlesWhenFiltering()
+        }
+    }
+    
+    private func noResultsTitlesWhenSearching() -> [PostListStatusFilter:String] {
+        
+        let draftMessage = String(format: NSLocalizedString("No drafts match your search for %@", comment: "The '%@' is a placeholder for the search term."), currentSearchTerm()!)
```

Comment:
In that case, maybe... we should move the opening brace `{` to a new line, always.

That way everything would be consistent, no matter what.